gui/test/cppchecklibrarydata/files/container_unhandled_element.cfg

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0"?>
 <def format="2">
   <container id="stdStack" startPattern="std :: stack &lt;" inherits="stdContainer">
-    <!-- error: invalid element acess -->
-    <acess>
```

Comment:
I think this was intentionally misspelled to test that cppcheck throws an error in this case. That's why the build fails.